tv_apps/smart-system/fxa/js/screens/fxam_enter_email.js

```diff
@@ -88,19 +89,35 @@ var FxaModuleEnterEmail = (function() {
     this.fxaEmailInput.addEventListener(
       'input',
       function onInput(event) {
+        if(this.fxaEmailInput.value) {
+          this.fxaEmailCleanBtn.classList.add('show');
+        } else {
+          this.fxaEmailCleanBtn.classList.remove('show');
+        }
         _enableNext(event.target);
-      }
+      }.bind(this)
     );
     this.fxaEmailInput.addEventListener(
       'focus',
       function onFocus() {
         this.fxaLogo.setAttribute('hidden', true);
+        if(this.fxaEmailInput.value) {
+          this.fxaEmailCleanBtn.classList.add('show');
+        }
       }.bind(this)
     );
     this.fxaEmailInput.addEventListener(
       'blur',
       function onBlur() {
         this.fxaLogo.removeAttribute('hidden');
+        this.fxaEmailCleanBtn.classList.remove('show');
+      }.bind(this)
+    );
+
+    this.fxaEmailCleanBtn.addEventListener(
+      'mousedown',
```

Comment:
We need to support key events as well. Please change it to ```click```, and check whether it's mouse left button or enter key.